src/kinematics/kdl/kdl_model.cpp

```diff
@@ -48,15 +48,15 @@ KDLModelTpl<S>::KDLModelTpl(const std::string &urdf_filename,
 template <typename S>
 std::tuple<VectorX<S>, int> KDLModelTpl<S>::chainIKLMA(size_t index,
                                                        const VectorX<S> &q0,
-                                                       const Vector7<S> &pose) const {
```

Comment:
Yeah, I'm actually not sure why KDL was here before.
But let's keep it for now since the benefits of removing it only include making the python wheel slightly smaller, imo.